resources/public/api/conf/1.0/schemas/contact-details-response.json

```diff
@@ -26,17 +26,20 @@
       "description": "Telephone numbers for the individual.",
       "properties": {
         "mobileTelephone": {
-          "type": "string",
+          "type": "array",
+          "items": { "type": "string" },
           "description": "The mobile telephone number for the individual.",
           "example": "07700 900000"
         },
         "daytimeTelephone": {
-          "type": "string",
+          "type": "array",
+          "items": { "type": "string" },
           "description": "The daytime telephone number for the individual.",
```

Comment:
May be worth pluralising as I imagine this may cause confusion as to why it is an array.

Reply:
Done